### PR TITLE
Document token operations

### PR DESCRIPTION
**Pull request by diversenok**
This pull request documents various operations on tokens, capabilities, and AppContainer SIDs.

### CODE REVIEW

**Comment by diversenok** [comment]
Also, do you think it might be useful to create a changelog page on the website to track updates to the documentation? Perhaps, a markdown file (rendered the same way as descriptions) where we can link to pages with substantial changes? Maybe even use some space on the main page to render the top portion of this changelog?

**Comment by m417z** [comment]
Good idea, I'm on a vacation, I'll look at it when I get home and have some spare time.

File: descriptions/ntcurrentprocess.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-This macro is [documented in Windows Driver Kit](https://learn.microsoft.com/en-us/windows-hardware/drivers/kernel/zwcurrentprocess). It is a native equivalent of the [`GetCurrentProcess`](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess) function and returns a pseudo-handle that grants `PROCESS_ALL_ACCESS` to the current process. You do not need to call `NtClose` on the returned handle.
+This macro is a native equivalent of the [`GetCurrentProcess`](https://learn.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getcurrentprocess) function. It returns a pseudo-handle that grants `PROCESS_ALL_ACCESS` to the current process. You do not need to call `NtClose` on the returned handle.
```
Comment (m417z) [inline descriptions/ntcurrentprocess.md:1]: Why did you remove the Windows Driver Kit link?

Comment (diversenok) [reply descriptions/ntcurrentprocess.md:1]: Hmm, that wasn't intentional. I think I accidentally rolled back some changes in this and `NtCurrentThread` descriptions while adding the "Applicable to" section. I'll add a fix shortly

**Comment by m417z** [comment]
> Also, do you think it might be useful to create a changelog page

Done:

image
